src/game.cpp

```diff
@@ -11556,14 +11623,21 @@ bool game::check_safe_mode_allowed( bool repeat_safe_mode_warnings )
     if( new_seen_mon.empty() ) {
         // naming consistent with code in game::mon_info
         spotted_creature_name = _( "a hostile survivor" );
+        get_safemode().whitelist = "human";
```

Comment:
Would be useful if you could also trigger it on neutral survivors. They can become hostile, steal stuff etc.